### ISSUE TITLE
[sign_in_with_apple] Execution failed for task ':sign_in_with_apple:compileDebugKotlin'

### ISSUE THREAD

**Issue by rjkirk** [aboutyou/dart_packages#289]
Created new flutter project just added `sign_in_with_apple: ^3.3.0` to pubspec. Flutter run fails with 
```
e: /Users/.../development/flutter/.pub-cache/hosted/pub.dartlang.org/sign_in_with_apple-3.3.0/android/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/SignInWithApplePlugin.kt: (23, 8): Class 'SignInWithApplePlugin' is not abstract and does not implement abstract member public abstract fun onActivityResult(p0: Int, p1: Int, p2: Intent): Boolean defined in io.flutter.plugin.common.PluginRegistry.ActivityResultListener
e: /Users/.../development/flutter/.pub-cache/hosted/pub.dartlang.org/sign_in_with_apple-3.3.0/android/src/main/kotlin/com/aboutyou/dart_packages/sign_in_with_apple/SignInWithApplePlugin.kt: (127, 3): 'onActivityResult' overrides nothing

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':sign_in_with_apple:compileDebugKotlin'.
```

Seemed similar to [#271](https://github.com/aboutyou/dart_packages/issues/271), however none of the suggestions worked and have different logs.

Flutter doctor output:

```
[✓] Flutter (Channel master, 2.11.0-0.0.pre.633, on macOS 11.5.2 20G95 darwin-x64, locale en-GB)
    • Flutter version 2.11.0-0.0.pre.633 at /Users/.../development/flutter
    • Upstream repository https://github.com/flutter/flutter.git
    • Framework revision 54a9d65282 (4 days ago), 2022-02-19 10:09:26 -0800
    • Engine revision f6ea6186ee
    • Dart version 2.17.0 (build 2.17.0-125.0.dev)
    • DevTools version 2.11.0

[✓] Android toolchain - develop for Android devices (Android SDK version 32.0.0)
    • Android SDK at /Users/.../Library/Android/sdk
    • Platform android-31, build-tools 32.0.0
    • Java binary at: /Applications/Android Studio.app/Contents/jre/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7590822)
    • All Android licenses accepted.

[✓] Xcode - develop for iOS and macOS (Xcode 13.2.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • CocoaPods version 1.11.2

[✓] Chrome - develop for the web
    • Chrome at /Applications/Google Chrome.app/Contents/MacOS/Google Chrome

[✓] Android Studio (version 2021.1)
    • Android Studio at /Applications/Android Studio.app/Contents
    • Flutter plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/9212-flutter
    • Dart plugin can be installed from:
      🔨 https://plugins.jetbrains.com/plugin/6351-dart
    • Java version OpenJDK Runtime Environment (build 11.0.11+0-b60-7590822)

[✓] Connected device (2 available)
    • Android SDK built for x86 (mobile) • emulator-5554 • android-x86    • Android 8.0.0 (API 26) (emulator)
    • Chrome (web)                       • chrome        • web-javascript • Google Chrome 98.0.4758.102

[✓] HTTP Host Availability
    • All required HTTP hosts are available

• No issues found!

```

**Comment by rjkirk** [comment]
Realised this is the issue fixed in pull request [#287](https://github.com/aboutyou/dart_packages/pull/287/), whose commit can be used by using 
```
sign_in_with_apple:
    git:
      url: git://github.com/aboutyou/dart_packages.git
      ref: df207317beab4110314ecd4dac05a8870ec22bb6
      path: packages/sign_in_with_apple/sign_in_with_apple
```

in the pubspec. Closing.

**Comment by sergiossm** [comment]
> Realised this is the issue fixed in pull request [#287](https://github.com/aboutyou/dart_packages/pull/287/), whose commit can be used by using
> 
> ```
> sign_in_with_apple:
>     git:
>       url: git://github.com/aboutyou/dart_packages.git
>       ref: df207317beab4110314ecd4dac05a8870ec22bb6
>       path: packages/sign_in_with_apple/sign_in_with_apple
> ```
> 
> in the pubspec. Closing.

Same issue here. I tried the above solution but not working for me. Any clue what might be happening?
